picard/coverart/providers/caa.py

```diff
@@ -36,6 +42,139 @@
     1: "large",
 }
 
+class CAATypesSelectorDialog(QtGui.QDialog):
+    _columns = 4
+
+    def __init__(self, parent=None, types=[]):
+        super(CAATypesSelectorDialog, self).__init__(parent)
+
+        self.setWindowTitle(_("Cover art types"))
+        self._items = {}
+        self.layout = QtGui.QVBoxLayout(self)
+
+        grid = QtGui.QWidget()
+        gridlayout = QtGui.QGridLayout()
+        grid.setLayout(gridlayout)
+
+        rows = len(CAA_TYPES) // self._columns + 1
+        positions = [(i, j) for i in range(rows) for j in range(self._columns)]
```

Comment:
Agreed, modified in https://github.com/musicbrainz/picard/commit/951e08afd77e09235ff4901b791de2acfd14b148